qemu/tests/cfg/tpm_verify_device.cfg

```diff
@@ -30,9 +35,6 @@
     variants:
         - with_emulator:
             tpm_type_tpm0 = emulator
-            tpm_model_tpm0 = tpm-crb
-            ppc64le, ppc64:
-                tpm_model_tpm0 = tpm-spapr
```

Comment:
Could you explain to me the purpose of move this part as the default param in the whole cfg?
Thanks.

Reply:
As I mentioned above, it is better to assign a default device for each architecture rather than specify tpm-crb.